### ISSUE TITLE
MessageConsole ClassCastException when deploying

### ISSUE THREAD

**Issue by chanseokoh** [GoogleCloudPlatform/gcloud-eclipse-tools#567]
When I deploy, I see the following error. However, the console shows deploy output normally.

```
!MESSAGE Problems occurred when invoking code from plug-in: "org.eclipse.ui.console".
!STACK 0
java.lang.ClassCastException: org.eclipse.ui.console.MessageConsole cannot be cast to com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsole
	at com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsolePageParticipant.init(LocalAppEngineConsolePageParticipant.java:31)
	at org.eclipse.ui.internal.console.ConsoleView$3.run(ConsoleView.java:329)
```

```
!ENTRY org.eclipse.ui.console 4 120 2016-08-16 11:59:48.082
!MESSAGE Error logged from Console plug-in: 
!STACK 0
java.lang.ClassCastException: org.eclipse.ui.console.MessageConsole cannot be cast to com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsole
	at com.google.cloud.tools.eclipse.appengine.localserver.ui.LocalAppEngineConsolePageParticipant.init(LocalAppEngineConsolePageParticipant.java:31)
	at org.eclipse.ui.internal.console.ConsoleView$3.run(ConsoleView.java:329)
```

**Comment by elharo** [comment]
I suspect this is related to, possibly the same as #571

**Comment by chanseokoh** [comment]
Closed by #582.
